client/src/components/LeftSidebar/components/Categorical/components/Category/components/AnnoDialogEditCategoryName/AnnoDialogEditCategoryName.tsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
```

Comment:
Rename dialog wraps the generic `LabelInput` and dispatches `annotationRenameCategoryAction`; closing the loop with autosave happens in the thunk.